cltools.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
Before I had this PR up, I was looking at homebrew packages for autotools. It looks like there are some incompatibilty issues filed for the latest version, so I just grabbed the latest using these scripts. I kept this in as a fallback in case things with brew don't work out.